packages/waku/src/lib/middleware/rsc/utils.ts

```diff
@@ -1,5 +1,14 @@
 // This file should not include Node specific code.
 
+// copy from Node.js source code
+const win32Sep = '\\';
+const posixSep = '/';
+
+// Refs: https://github.com/rollup/plugins/blob/d49bbe8dc5ec41157de5787c72c858f73be107ff/packages/pluginutils/src/normalizePath.ts
+export function normalizePath(filePath: string) {
+  return filePath.split(win32Sep).join(posixSep);
```

Comment:
That's simple.
I'd inline them, as it's fairly readable.
```suggestion
  return filePath.split('\\').join('/');
```

This doesn't work if posix path contains backslashes. Can't be helped.